main/ertp/guide/issuers-and-mints.md

```diff
@@ -38,95 +38,46 @@ API Reference](/ertp/api/#ertp-api).
 
 - **Create issuer operation**
   - [`makeIssuerKit(allegedName, assetKind, displayInfo=)`](/ertp/api/issuer.md#makeissuerkit-allegedname-assetkind-displayinfo)
-  - Makes an `issuer` and its related `mint` and `brand`.
-    Returns ` { mint, issuer, brand }` The `mint` and
-    `brand` are in unchangeable one-to-one relationships with the `issuer`
-    and each other. 
-    
-    The `allegedName` is available from the `brand` to describe assets, but should not
-    be trusted. 
-    
-    `assetKind` specifies if the associated asset is of kind `AssetKind.NAT` (`nat`) 
-    (the default value) or `AssetKind.SET` (`set`);
-    see the [`AmountMath` page](./amount-math.md) for details. 
-    
-    `displayInfo` is the number of places to the right of the decimal point to display of any
-    `values` associated with the created `brand`. It defaults to `undefined`
-    
+    - Create and return a new `issuer` and its associated `mint` and `brand`.
     - <<< @/snippets/ertp/guide/test-issuers-and-mints.js#import
       <<< @/snippets/ertp/guide/test-issuers-and-mints.js#makeIssuerKit
 - **Get information about the issuer operations**
-  - [`issuer.getBrand()`](/ertp/api/issuer.md#issuer-getbrand)
-    - Returns the `brand` the `issuer` is in a one-to-one relationship with. The `brand` is not closely
-      held, so it can be used by fake digital assets and `amounts`. Do
-      not trust this method alone to ensure the `issuer` has the right `brand`.
-    - <<< @/snippets/ertp/guide/test-issuers-and-mints.js#getBrand
   - [`issuer.getAllegedName()`](/ertp/api/issuer.md#issuer-getallegedname)
-    - Returns the `issuer`/`mint`'s
-      [`allegedName`](/glossary/#allegedname),
-	the non-trusted human-readable name of the `issuer`'s associated `brand`.
+    - Return the `allegedName` for the `issuer` (the non-trusted human-readable name of its associated `brand`).
     - <<< @/snippets/ertp/guide/test-issuers-and-mints.js#getAllegedName
   - [`issuer.getAssetKind()`](/ertp/api/issuer.md#issuer-getassetkind)
-    - Get the kind of asset for this `issuer`, either `AssetKind.NAT` (`nat`),
-      or `AssetKind.SET` (`set`).
+    - Return the kind of the `issuer`'s asset; either `AssetKind.NAT` ("nat") or `AssetKind.SET` ("set").
     - <<< @/snippets/ertp/guide/test-issuers-and-mints.js#getAssetKind
+  - [`issuer.getBrand()`](/ertp/api/issuer.md#issuer-getbrand)
+    - Return the `brand` for the `issuer`.
+    - <<< @/snippets/ertp/guide/test-issuers-and-mints.js#getBrand
 - **Purse operation**
   - [`issuer.makeEmptyPurse()`](/ertp/api/issuer.md#issuer-makeemptypurse)
-    - Returns a new empty `purse` for the `brand` associated with the `issuer`. The `purse` only accepts valid 
-      deposits of its associated `brand`, so you can retroactively identify a valid `payment` of that `brand`
-      by successfully depositing it into this `purse`.
+    - Make and return an empty `purse` that holds assets of the `brand` associated with the `issuer`.
```

Comment:
```suggestion
    - Make and return an empty `purse` for holding assets of the `brand` associated with the `issuer`.
```
Since it is empty, I think this is slightly better.